pandas/core/internals/managers.py

```diff
@@ -145,19 +145,20 @@ def __init__(
 
         self._rebuild_blknos_and_blklocs()
 
-    def make_empty(self, axes=None):
+    def make_empty(self, axes=None) -> "BlockManager":
         """ return an empty BlockManager with the items axis of len 0 """
         if axes is None:
-            axes = [ensure_index([])] + [ensure_index(a) for a in self.axes[1:]]
+            axes = [Index([])] + self.axes[1:]
```

Comment:
What error was this fixing?

Reply:
Not an error, just unnecessarily complicated